manifest.yml

```diff
@@ -1,11 +1,12 @@
 name : "coreHTTP"
 version: "v2.1.0"
-description: |
-  "Client implementation of the HTTP/1.1 specification for embedded devices.\n"
+description: "Client implementation of the HTTP/1.1 specification for embedded devices."
+license: "MIT"
+
 dependencies:
   - name : "llhttp"
     version: "release/v6.0.5"
+    license: "MIT"
     repository:
       type: "git"
-      url: "https://github.com/nodejs/llhttp"
-license: "MIT"
+      url: "https://github.com/nodejs/llhttp.git"
```

Comment:
Why was .git added here?

Reply:
".git" was added by Paul for the corePKCS11 manifest.yml file, so for consistency I added ".git" for the rest of the libraries as well.